tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_redis_22407.go

```diff
@@ -27,11 +27,11 @@ func (w workaroundRedis22407) Process(apiDefinition importerModels.AzureApiDefin
 
 			if _, ok := model.Fields["NotifyKeyspaceEvents"]; !ok {
 				model.Fields["NotifyKeyspaceEvents"] = sdkModels.SDKField{
-					Required:    false,
-					ReadOnly:    false,
-					Sensitive:   false,
-					JsonName:    "notify-keyspace-events",
-					Description: "The KeySpace Events which should be monitored.",
+					Required:  false,
+					ReadOnly:  false,
+					Sensitive: false,
+					JsonName:  "notify-keyspace-events",
+					//Description: "The KeySpace Events which should be monitored.", // TODO: avoid flapping diff in api definitions, see https://github.com/hashicorp/pandora/issues/3325
```

Comment:
I don't think this needs to be commented out since it's a workaround and we're adding this data instead of attempting to parse it from the spec

Reply:
Cool, I've undone that 👍